charts/hocs-network-policies/templates/hocs-docs-networkpolicy.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: hocs-docs-policy
+spec:
+  podSelector:
+    matchLabels:
+      name: hocs-docs
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+        - podSelector:
```

Comment:
Can't remember if FE communicates directly with docs?

Reply:
it does